src/cozmo/anim.py

```diff
@@ -89,10 +95,10 @@ def _encode(self):
             robotID=self.robot.robot_id, trigger=self.trigger.id, numLoops=self.loop_count)
 
     def _dispatch_completed_event(self, msg):
-        self.dispatch_event(EvtAnimationCompleted,
+        self._completed_event = EvtAnimationCompleted(
```

Comment:
With the change to this method, how is the behavior different than it was before?